scripts/upgrade_admin_server.sh

```diff
@@ -8,27 +8,42 @@
 
 LOGFILE=/var/log/crowbar/admin-server-upgrade.log
 mkdir -p "`dirname "$LOGFILE"`"
-
-# Copy stdout to fd 3
-exec 3>&1
-# Create fd 4 for logfile
-exec 4>> "$LOGFILE"
-# Set fd 1 and 2 to logfile
-exec 1>&4 2>&1
+exec >>"$LOGFILE" 2>&1
```

Comment:
yep (although, the script is actually not packaged in crowbar package for Cloud7)